app/views/layouts/_student_footer.html.erb

```diff
@@ -9,7 +9,7 @@
         <div class="w-full flex flex-row flex-wrap text-xs font-semibold">
           <% presenter.nav_links.each do |nav_link| %>
             <div class="w-1/3">
-              <%= link_to nav_link[:title], nav_link[:url], class: "no-underline pt-2 pb-1 block hover:text-primary-500" %>
+              <%= link_to nav_link[:title], nav_link[:url], { class: 'footer-column__link' }.merge(nav_link[:target_blank] || {} ) %>
```

Comment:
1. The `footer-column__link` class doesn't work here - that's from the older Bootstrap-based stylesheet which isn't used on any of the newer pages. In fact, I think practically all of the older Bootstrap-based pages are inaccessible right now - so this is the only footer partial that matters. 🤔 
2. Decide on attributes here, based on `custom` property:

```erb
<%= link_to(
      nav_link[:title], nav_link[:url], {
      class: 'no-underline pt-2 pb-1 block hover:text-primary-500'
    }.merge(nav_link[:custom] ? { target: '_blank', rel: 'noopener' } : {})
    )
%>
```